package.json

```diff
@@ -26,6 +26,7 @@
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "lodash": "^4.17.10",
+    "semver": "^5.5.0",
```

Comment:
why not using strict version of the dependency?

Reply:
Most of the dependencies don't use strict version. I consider converting all of them to a strict version and adding a package-lock.json but in separate PR.